deploy/Provisioning-1674548289785.json

```diff
@@ -2149,9 +2149,9 @@
     "list": [
       {
         "current": {
-          "selected": true,
-          "text": "crcp01ue1-prometheus",
-          "value": "crcp01ue1-prometheus"
+          "selected": false,
+          "text": "crcs02ue1-prometheus",
+          "value": "crcs02ue1-prometheus"
         },
```

Comment:
This happens when you store JSON when the app is switched on prod/stage. Stage is perhaps better.